### DISCUSSION TITLE
Are there any forseeable problems with running two instances of isso on the same server?

### DISCUSSION THREAD

**Discussion by abdullahkhalids** [posativ/isso#956] [Q&A] [answered]
I have two blogs on my website, for which I want to have separate isso instances. I already have one instance installed and running on my server and it works well. I want to install the second instance on the same server. 

My plan is to create a second virtual environment for the second isso. And just have different `listen` and `public-endpoint` for the second instance. Nginx proxy settings for the second instance also seem straightforward.

Are there any foreseeable problems I am missing? I would like to know before I start messing with my server.

**Comment by ix5** [comment, marked as answer]
As long as you set up your proxy correctly, everything should be fine.
In fact, Isso is designed to be run for only on website/domain at a time, requiring [multiple instances](https://isso-comments.de/docs/reference/multi-site-sub-uri/) if you want to serve multiple websites.

While at some point in the future we might want to unify all sites under one Isso instance, at the moment, your approach is exactly the correct one.

**Comment by abdullahkhalids** [comment]
Thank you
